SPECS/kernel/config

```diff
@@ -7857,6 +7861,7 @@ CONFIG_STRIP_ASM_SYMS=y
 # CONFIG_DEBUG_SECTION_MISMATCH is not set
 CONFIG_SECTION_MISMATCH_WARN_ONLY=y
 # CONFIG_DEBUG_FORCE_FUNCTION_ALIGN_64B is not set
+CONFIG_ARCH_WANT_FRAME_POINTERS=y
```

Comment:
result of: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/1d95dbaa6ee708779af0cc4f642a6634e8703269